src/components/Card/Card.js

```diff
@@ -7,6 +7,12 @@ function Card(props) {
             <Statistic type="Breed" details={props.name}/>
             <Statistic type="Height" details={props.height}/>
             <Statistic type="Temperament" details={props.temperament}/>
+            <button className="like-button" onClick={()=>{
```

Comment:
We can probably do this opening element in one line - also can we have spaces either side of the `=>`?

Reply:
on it